src/wovnio/html/HtmlConverter.php

```diff
@@ -280,6 +280,16 @@ private function insertHreflangTags($html)
             array_push($hreflangTags, '<link rel="alternate" hreflang="' . Lang::iso6391Normalization($lang_code) . '" href="' . $href . '">');
         }
 
+        if (isset($this->store->settings['hreflang_x_default_lang'])) {
+            $x_default_hreflang_regex = "/<link [^>]*hreflang=[\"']?(x-default)[\"']?(\s[^>]*)?\>/iU";
```

Comment:
I prefer the following regex. I think it works as same way.
```
<link[^>]*hreflang=[\"']?x-default[\"']?[^>]*>
```
- No need to do capture
- `[^>]` can include space
- `*` can be nothing, so no need to add `?`

Reply:
Fixed in https://github.com/WOVNio/WOVN.php/pull/207/commits/40443223bc9f64743e846c214cbcd596f871688b.